meshack-mbuvi/ride-my-way-api
# #158804740 restructure app to remove duplicates on files responsible for initialization of database

#### What does this PR do?
This pul request restructures code to remove duplicates.
It also models database connection and table creation in a single file

#### Description of Task to be completed?
- Delete code for table creation in application/models/__init__.py
- Have all database functionality in manage.py
- Update instance/config.py file to save database credentials to environmental variables.
- In manage.py file, read database credentials from the environmental variables

#### How should this be manually tested?
using your terminal,
- clone this repo
- `cd ride-my-way`
- set and activate your virtual environment
- `git checkout ch-158804740`
- Run `nosetests -v` to confirm all tests run successfully
- Run `python3 run.py`

**Pivotal tracker stories**
[Restructure code to remove duplicates #158804740](https://www.pivotaltracker.com/story/show/158804740)

**How to review**
Scrutinize manage.py for the logic used, instance/config.py for the configuration management
